### DISCUSSION TITLE
How to use k=25 for testing?

### DISCUSSION THREAD

**Discussion by ischicopter** [swar/Swar-Chia-Plot-Manager#382] [General]
Hi everyone. What a fantastic piece of software. I have been looking for exactly this for days now.

However I am running into trouble when trying to set k=25 for testing purposes. k32 works fine.

The logs say: 

"
k=32 is the minimun size for farming.
If you are testing and you want to use smaller size please add the --override-k flag.
"

Where exactly do I have to add this override flag ? 

Thank you very much in advance

cheers
Frank

**Comment by emreyahsi** [comment]
I am looking for the answer too.  if there is no solution for it. Maybe a job parameter can be added next versions.
